e2e/velero/cli.go

```diff
@@ -65,3 +69,16 @@ aws_secret_access_key=%s
 		accessKey, secretKey,
 	)), 0644)
 }
+
+func patchNodeAgentDaemonset(kubeconfig string) (*gexec.Session, error) {
```

Comment:
this comes from velero docs on openshift recommendations https://velero.io/docs/v1.9/restic/

Reply:
wouldn't this affect _all_ tests though? is there a way to only enable this for openshift?

Reply:
sounds good. can add some additional logic to e2e enable for OpenShift only